src/Mirza.Web/Pages/Index.cshtml.cs

```diff
@@ -90,8 +108,8 @@ public async Task<IActionResult> OnPostAsync()
                     Description = Input.Description ?? "-",
                     Details = Input.Details ?? "-",
                     EntryDate = Input.Date,
-                    StartTime = Input.Start,
-                    EndTime = Input.End
+                    StartTime = TimeSpan.Parse(Input.Start),
+                    EndTime = TimeSpan.Parse(Input.End)
```

Comment:
since `Input.Start` and `Input.End` are now strings, `TryParse` would be a better way to go

Reply:
Fixed